backend/uclapi/uclapi/settings.py

```diff
@@ -23,7 +23,7 @@
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = os.environ.get("SECRET_KEY")
-
+CACHET_KEY = os.environ.get("CACHET_TOKEN")
```

Comment:
Any reason we don't juse call this variable `CACHET_TOKEN`?

Reply:
Nope I'm just bad at naming things :man_shrugging: